client/src/components/Homepage/index.js

```diff
@@ -5,6 +5,8 @@ import React, { useEffect, useState } from 'react'
 import AceEditor from 'react-ace'
 import axios from 'axios'
 import openSocket from 'socket.io-client'
+import { AiFillGithub,FiClock,FiFileText,FiRotateCcw,FiAlignJustify } from "react-icons/ai";
```

Comment:
@kailash360 A mistake has been made here while refactoring. This line would throw an error. Please fix this.

Reply:
I have fixed this now. Sorry for the silly mistake 😅.